src/IECore/FileSequenceParameter.cpp

```diff
@@ -130,7 +131,7 @@ bool FileSequenceParameter::valueValid( const Object *value, std::string *reason
 
 	if ( m_extensions.size() )
 	{
-		std::string ext = boost::filesystem::extension( boost::filesystem::path( fileSequence->getFileName() ) );
+		std::string ext = boost::filesystem::path( boost::filesystem::path( fileSequence->getFileName() ) ).extension().string();
```

Comment:
Unless I'm missing something, I think there is an unnecessary call to `path()` here as well?